### ISSUE TITLE
Bruno CLI exists successfully on errors

### ISSUE THREAD

**Issue by ahoka** [usebruno/bruno#618]
It should return a non-zero exit code on failure to run collections or after failed runs.

$ bru run
You can run only at the root of a collection
$ echo $?
0

**Comment by ynikolenko** [comment]
You haven't solved that problem yet?
I have the same problem, Windows 10, VS Code, trying to run one file.

PS С:\bruno> bru run '.\filename.bru'
You can run only at the root of a collection
PS С:\bruno>

**Comment by andrewflierman** [comment]
Make sure you have the bruno.json from where you running

**Comment by frederikb** [comment]
Yes, your current working directory must be the root of the collection:
```javascript
// todo
// right now, bru must be run from the root of the collection
// will add support in the future to run it from anywhere inside the collection
```
This is an unfortunately still open todo.

But this is not the topic of this issue. The problem is that this error is not communicated as such via the exit code of `bru`. Our test automation will falsely assume that the test succeeded when it has been wrongly set up.

**Comment by ryanotella** [comment]
This appears to be fixed now.

```
$ bru run
You can run only at the root of a collection

$ echo $?
4
```

**Comment by manuelnagler** [comment]
Can confirm. This also happens when there is an error due to connection refused:

`
test-single-request (connect ECONNREFUSED <redacted>)
Requests:    0 passed, 1 error, 1 total
Tests:       0 passed, 0 total
Assertions:  0 passed, 0 total
Exit code: 0
`

This makes bru basically unusable due to fatal errors leading to successful tests.

**Comment by lohit-bruno** [comment]
this issue has been resolved. please upgrade to the latest version of `bruno-cli` [v2.6.1].

the process exits with a non-zero code in the event of a network error or test failure.
